src/jit/compiler.cpp

```diff
@@ -540,7 +540,7 @@ var_types    Compiler::argOrReturnTypeForStruct(unsigned size, CORINFO_CLASS_HAN
         break;
 
     default:
-#if FEATURE_MULTIREG_RET
+#if FEATURE_MULTIREG_RET && !defined(_TARGET_X86_)
```

Comment:
Actually, most of this code doesn't make sense to me in the first place since FEATURE_MULTIREG_RET isn't set for _TARGET_ARM64_, so except for the ifs, none of this ever gets executed. The reason I had to add the !_TARGET_X86_ is because MAX_RET_MULTIREG_BYTES isn't set for x86. I could update that (to be 0?) and then this code wouldn't change.